src/test/java/io/specto/hoverfly/ruletest/CaptureOrSimulationModeTest.java

```diff
@@ -0,0 +1,28 @@
+package io.specto.hoverfly.ruletest;
+
+import io.specto.hoverfly.junit.core.HoverflyMode;
+import io.specto.hoverfly.junit.rule.HoverflyRule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CaptureOrSimulationModeTest {
+
+    @Test
+    public void shouldInstantiateHoverflyInCaptureModeInCaseOfNoRecord() {
+
+        final HoverflyRule hoverflyRule = HoverflyRule.inCaptureOrSimulationMode("mynewservicerecord.json");
```

Comment:
Exactly I think that for this "mode" this test is enough since it imply reuses already created modes. No worries.